app/assets/stylesheets/_custom.scss

```diff
@@ -20,6 +20,10 @@ html {
   font-size: 20px;
 }
 
+.header-with-covid-banner-spacer {
+  margin-top: ($spacer * 7.25);
+}
+
```

Comment:
My thought was this would become .header-spacer when we remove the covid banner.